mptt/models.py

```diff
@@ -341,6 +342,17 @@ def register(meta, cls, **kwargs):
                     if field_name not in existing_field_names:
                         field = models.PositiveIntegerField(db_index=True, editable=False)
                         field.contribute_to_class(cls, field_name)
+                        
+                if django.VERSION >= (1, 11):
```

Comment:
With recent changes, you can drop this guard. django-mptt now only suppors 1.11+.